CHANGES.md

```diff
@@ -8,8 +8,16 @@ Release Notes.
 
 ### Features
 
+- Support `indexed_only` flag to the tag specification.
+- Support `Analyzer` to the index rule.
+- Add `exist` endpoints to the metadata registry.
+- Set `CompletableFuture<Void>` to the return type of write processor.
```

Comment:
@kezhenxu94 could you recheck this part? Whether is it suitable for OAP writing and self-obs?

Reply:
I think this is good in OAP side